README.md

```diff
@@ -34,7 +34,7 @@ repositories {
 Add the following dependencies to your `build.gradle` (in your app module) in the `dependencies` block:
 
 ```gradle
-bpkVersion = "36.3.0"
+bpkVersion = "x.x.x"
```

Comment:
Shall we remove the variable from the sample and just use the symbols in the version instead?